### DISCUSSION TITLE
come back to edureka class

### DISCUSSION THREAD

**Discussion by mohammedawez** [Jay-study-nildana/FrontEndForStudents#21] [General]
@Jay-study-nildana 

its me md awez
sir i want u as teacher in edureka classes other wise i ill not join the classes, and i ill cancel the course also

#comebacktoedurekwewantuasteacher

**Comment by Jay-study-nildana** [comment]
@mohammedawez 

thank you for you kind words brotha. 

Don't cancel the course. Finish it, and become a developer and have a nice career.

Jay
see my memes here, https://imgflip.com/user/codingtutorjay
join my open source community, here, https://github.com/BigbabeeSmallbabee
insta and twitter and other things, available on my github profile.

**Reply by mohammedawez** [reply]
sir u will not cm back wt???  i m not intrested in classs, come n take class again !!!!!!!!!!!!!! i m not understanding anything, your classes were crazy n good.


@Jay-study-nildana

**Reply by Jay-study-nildana** [reply]
@mohammedawez 

a new tutor has already started. don't leave the class, finish the classes and get your certificate. 

and this github will always be open. feel free to ask code doubts and take help me from, other students and from the community.

all the best brotha.

Jay
see my memes here, https://imgflip.com/user/codingtutorjay
join my open source community, here, https://github.com/BigbabeeSmallbabee
insta and twitter and other things, available on my github profile.
